setup.py

```diff
@@ -7,7 +7,7 @@
 
 setuptools.setup(
     name="slicedimage",
-    version="1.0.4",
+    version="2.0.0",
```

Comment:
Lol this is unfortunate. Package version is +1 of the format version? :-)

Reply:
yeah, I've always found having a library that has both an API and a file format version leads to some confusion. :(